js_test.js

```diff
@@ -0,0 +1 @@
+console.log("foo")
```

Comment:
Use the global form of "use strict".<br>Definition for rule 'keyword-spacing' was not found<br>Unexpected console statement.<br>Strings must use singlequote.